sdk/entrypoint/src/lib.rs

```diff
@@ -17,7 +18,11 @@ use {
         slice::{from_raw_parts, from_raw_parts_mut},
     },
 };
-pub use {solana_account_info::MAX_PERMITTED_DATA_INCREASE, solana_program_error::ProgramResult};
+// need to re-export msg for custom_heap_default macro
+pub use {
+    solana_account_info::MAX_PERMITTED_DATA_INCREASE, solana_msg::msg as __msg,
```

Comment:
nit: Is there a reason for the double underscore vs just one? We don't seem to use the double-underscore anywhere for private members, so how about just `_msg`?

Reply:
It indicates that something is technically public but shouldn't be used externally https://users.rust-lang.org/t/when-to-name-a-mod-starting-with-2-underscore/85953

Reply:
Gotcha, works for me!